test/integration/setup.js

```diff
@@ -72,18 +73,8 @@ const populate = (container, [ head, ...tail ] = fixtures) =>
 // this hook won't run if `test-unit` is called, as this directory is skipped
 // in that case.
 const initialize = async () => {
-  const migrator = knexConnect(config.get('test.database'));
-  const { log } = console;
-  try {
-    await migrator.raw('drop owned by current_user');
-    // Silence logging from migrations.
-    console.log = noop; // eslint-disable-line no-console
-    await migrator.migrate.latest({ directory: appRoot + '/lib/model/migrations' });
-  } finally {
-    console.log = log; // eslint-disable-line no-console
-    await migrator.destroy();
-  }
-
+  await withPg(dbConfig, client => client.query('DROP OWNED BY CURRENT_USER'));
+  execSync('make migrations', { env: { ...process.env, NODE_CONFIG: JSON.stringify({ default: { database: dbConfig } }) } });
```

Comment:
Could this change be made for the existing migrator?

Are there implications for test performance?

Reply:
affects tests in:

```
test/integration/api/datasets.js
test/integration/api/entities.js
test/integration/api/offline-entities.js
test/integration/api/submissions.js
test/integration/other/encryption.js
test/integration/other/migrations.js
test/integration/other/transactions.js
test/integration/task/s3.js
test/integration/worker/worker.js
```

Reply:
Single run:

method | execution time / s
---|---
`migrator.migrate.latest()` | `335s`
`make migrations` | `342s`

Reply:
execSync() seems to be consistently 20s slower:

![Screenshot_2025-01-18_09-42-21](https://github.com/user-attachments/assets/9bb609da-cc02-4eb6-b065-9887661c5697)